src/tracer.js

```diff
@@ -55,6 +73,7 @@ export default class Tracer {
      * @return {Span} - a new Span object.
      */
     startSpan(nameOrFields, fields) {
```

Comment:
since we're breaking the API already, why can't we make it strict, `startSpan(operationName, options)`? It will be consistent with APIs in other languages. The current flexibility seems very superficial, requiring more work from implementations.